ntfs_fr.md

```diff
@@ -357,18 +357,19 @@ stop_headsign | chaine | Optionnel |
 pickup_type | entier (1) | Optionnel | Indication sur l'horaire (issues du gtfs)
 drop_off_type | entier (1) | Optionnel | Indication sur l'horaire (issues du gtfs)
 local_zone_id  | entier | Optionnel | identifiant de la zone d'ITL de l'horaire
-date_time_estimated | entier (2) | Optionnel | Précise si l'heure de passage est fiable ou si elle est donnée à titre indicative
+stop_time_type | entier (2) | Optionnel | Précise si l'heure de passage est fiable ou si elle est donnée à titre indicative
```

Comment:
How about stop_time_precision? (or exactness?)
I find "type" often too broad.

Reply:
We decided on `type` because this seemed coherent with what is done in GTFS generally (`route_type`, `location_type`, `pathway_type`, `pickup_type`, drop_off_type`, etc.). That said, it was also because of lack of a better name :blush: